.gitignore

```diff
@@ -5,3 +5,4 @@
 
 node_modules
 coverage
+package-lock.json
```

Comment:
https://docs.npmjs.com/files/package-lock.json npm says otherwise, it even appears in the console, but I don't think anyone really wants to commit this fine